kotest-framework/kotest-framework-multiplatform-plugin-gradle/build.gradle.kts

```diff
@@ -92,6 +93,19 @@ gradlePlugin {
    }
 }
 
+kotlin {
```

Comment:
This module doesn't use convention plugin for some reason, so I copied the configuration here assuming that using convention plugin here is tricky for some reason